internal/extproc/translator/openai_gcpanthropic.go

```diff
@@ -148,14 +150,15 @@ func translateOpenAItoAnthropicTools(openAITools []openai.Tool, openAIToolChoice
 
 			// The parameters for the function are expected to be a JSON Schema object.
 			// We can pass them through as-is.
-			toolParam.InputSchema = anthropic.ToolInputSchemaParam{
-				Properties: openAITool.Function.Parameters,
-				// TODO: support extra fields.
-				ExtraFields: nil,
+			if openAITool.Function.Parameters != nil {
```

Comment:
the type is any, so should have a check here

Reply:
also when i ran integration tests, the json schema was invalid and throwing errors, this section is the fix for that.